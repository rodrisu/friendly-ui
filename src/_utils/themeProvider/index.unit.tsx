import React from 'react'

import { render, screen } from '@testing-library/react'

import { ThemeProvider } from './index'

describe('ThemeProvider', () => {
  it('renders properly', () => {
    render(<ThemeProvider>Some children content</ThemeProvider>)
    screen.getByText('Some children content')
    expect(document.head).toMatchSnapshot()
  })
})
