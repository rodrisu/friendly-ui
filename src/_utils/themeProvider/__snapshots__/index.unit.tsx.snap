// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeProvider renders properly 1`] = `
<head>
  <style
    data-styled=""
    data-styled-version="4.4.1"
  >
    
    
/* sc-component-id: sc-global-424256023 */
html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video,button,input,option,select,textarea{margin:0;padding:0;border:0;font-size:100%;font:inherit;vertical-align:baseline;-webkit-font-smoothing:antialiased;} body{line-height:1;color:#054752;} ol,ul{list-style:none;} table{border-collapse:collapse;border-spacing:0;} a{color:#00AFF5;-webkit-text-decoration:none;text-decoration:none;-webkit-tap-highlight-color:transparent;} *,*::after,*::before{box-sizing:border-box;}
    
/* sc-component-id: sc-global-3795229435 */
@font-face{font-family:'gt-eesti';font-weight:400;src:url('https://cdn.blablacar.com/kairos/assets/build/fonts/GT-Eesti-Pro-Display-Regular.woff2') format('woff2'), url('https://cdn.blablacar.com/kairos/assets/build/fonts/GT-Eesti-Pro-Display-Regular.woff') format('woff');font-style:normal;} @font-face{font-family:'gt-eesti';font-weight:500;src:url('https://cdn.blablacar.com/kairos/assets/build/fonts/GT-Eesti-Pro-Display-Medium.woff2') format('woff2'), url('https://cdn.blablacar.com/kairos/assets/build/fonts/GT-Eesti-Pro-Display-Medium.woff') format('woff');font-style:normal;} html{font-family:gt-eesti,'Helvetica Neue',Helvetica,Arial,sans-serif;quotes:'“' '”';-webkit-text-size-adjust:100%;} :lang(de){quotes:'„' '“';} :lang(fr){quotes:'« ' ' »';} :lang(es-ES){quotes:'«' '»';} :lang(hu){quotes:'„' '“';} :lang(pl){quotes:'„' '“';} :lang(ru){quotes:'« ' ' »';}
  </style>
</head>
`;
